component/web/security/pom.xml

```diff
@@ -61,6 +61,10 @@
       <groupId>org.gatein.captcha</groupId>
       <artifactId>simplecaptcha</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.bouncycastle</groupId>
+      <artifactId>bcprov-jdk15on</artifactId>
+    </dependency>
```

Comment:
Can you please move this into module `exo.portal.component.api` since it's a generic dependency used by multiple modules.

```suggestion
```

Reply:
Done